src/itemdef.h

```diff
@@ -49,6 +49,23 @@ enum ItemType
 	ITEM_TOOL,
 };
 
+enum class TouchControlMode
```

Comment:
(not sure what the class does here anyway)
```suggestion
enum TouchControlMode : u8
```

Reply:
[Scoped enumeration](https://en.cppreference.com/w/cpp/language/enum#Scoped_enumerations)